fairlearn/_input_validation.py

```diff
@@ -54,9 +54,9 @@ def _validate_and_reformat_input(X, y=None, expect_y=True, enforce_binary_labels
     if y is not None:
         # calling check_X_y with a 2-dimensional y causes a warning, so ensure it is 1-dimensional
         if isinstance(y, np.ndarray) and len(y.shape) == 2 and y.shape[1] == 1:
-            y = y.squeeze()
+            y = y.reshape(-1)
```

Comment:
why not squeeze anymore?

Reply:
it reduced the number of dimensions too much, down to a scalar, instead down to a one-dimensional array.